packages/react/src/components/Button/__snapshots__/Button.deprecated.test.tsx.snap

```diff
@@ -42,6 +42,7 @@ exports[`Button renders CompoundButton correctly 1`] = `
         content: "";
         left: 2px;
         outline: 1px solid #605e5c;
+        pointer-events: none;
```

Comment:
So, we recently made a change in an app to tell Fluent to show its focus rectangles on the initial load of the app instead of waiting for the first keyboard event. This was made in response to accessibility demands from some partners.
When we made this change, we learned that customers suddenly had to double-click various widgets in order to activate them, because the first click would be intercepted by the focus rectangle itself. Of course, said click would then trigger Fluent to hide the rectangle, but then the user would need to click again.

So it's unlikely this is a *recent* regression in Fluent, but it certainly is a *bug*.